xml/Microsoft.Extensions.Logging.Console/ConsoleFormatterOptions.xml

```diff
@@ -51,7 +51,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Includes scopes when <see langword="true" />.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <summary>Gets or sets whether scopes should be included or not.</summary>
        <value><see langword="true" /> when scopes should be included; <see langword="false" /> otherwise.</value>
```